MapboxNavigation/FeedbackViewController.swift

```diff
@@ -305,3 +326,225 @@ extension String {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    public override var sections: [FeedbackItem]  {
+        get {
+            guard let activeFeedbackType = activeFeedbackType else { return [] }
+            switch activeFeedbackType {
+            case .general:
+                return []
+            case .incorrectVisual(_):
+                return [FeedbackType.incorrectVisual(subtype: .turnIconIncorrect),
+                        FeedbackType.incorrectVisual(subtype: .streetNameIncorrect),
+                        FeedbackType.incorrectVisual(subtype: .instructionUnnecessary),
+                        FeedbackType.incorrectVisual(subtype: .instructionMissing),
+                        FeedbackType.incorrectVisual(subtype: .maneuverIncorrect),
+                        FeedbackType.incorrectVisual(subtype: .exitInfoIncorrect),
+                        FeedbackType.incorrectVisual(subtype: .laneGuidanceIncorrect),
+                        FeedbackType.incorrectVisual(subtype: .roadKnownByDifferentName),
+                        FeedbackType.incorrectVisual(subtype: .other)].map { $0.generateFeedbackItem() }
+            case .confusingAudio(_):
+                return [FeedbackType.confusingAudio(subtype: .guidanceTooEarly),
+                        FeedbackType.confusingAudio(subtype: .guidanceTooLate),
+                        FeedbackType.confusingAudio(subtype: .pronunciationIncorrect),
+                        FeedbackType.confusingAudio(subtype: .roadNameRepeated),
+                        FeedbackType.confusingAudio(subtype: .other)].map { $0.generateFeedbackItem() }
+            case .routeQuality(_):
+                return [FeedbackType.routeQuality(subtype: .routeNonDrivable),
+                        FeedbackType.routeQuality(subtype: .routeNotPreferred),
+                        FeedbackType.routeQuality(subtype: .alternativeRouteNotExpected),
+                        FeedbackType.routeQuality(subtype: .routeIncludedMissingRoads),
+                        FeedbackType.routeQuality(subtype: .other)].map { $0.generateFeedbackItem() }
+            case .illegalRoute(_):
+                return [FeedbackType.illegalRoute(subtype: .routedDownAOneWay),
+                        FeedbackType.illegalRoute(subtype: .turnWasNotAllowed),
+                        FeedbackType.illegalRoute(subtype: .carsNotAllowedOnStreet),
+                        FeedbackType.illegalRoute(subtype: .turnAtIntersectionUnprotected),
+                        FeedbackType.illegalRoute(subtype: .other)].map { $0.generateFeedbackItem() }
+            case .roadClosure(_):
+                return [FeedbackType.roadClosure(subtype: .streetPermanentlyBlockedOff),
+                        FeedbackType.roadClosure(subtype: .roadMissingFromMap),
+                        FeedbackType.roadClosure(subtype: .other)].map { $0.generateFeedbackItem() }
+            }
+        }
+    }
+
+    override var draggableHeight: CGFloat {
+        return 400
```

Comment:
I think using hardcoded height constant prevents showing `reportIssueLabel` at the top of the screen in landscape mode. Maybe it'd be better to use `UIScreen` height for landscape and `400` for portrait?
```suggestion
if UIDevice.current.orientation == .portrait {
    return 400
}

return UIScreen.main.bounds.height
```

Reply:
Just tested on iPad as well and seems that we should also handle additional cases there. E.g. does it make sense to show `FeedbackSubtypeViewController` full screen in both landscape and portrait there to have more items visible on screen?

Reply:
I think you are right that fullscreen is best. More simple code-wise and also better for possibly long lists.